spec/rspec/core/formatters_spec.rb

```diff
@@ -106,6 +106,21 @@ module RSpec::Core::Formatters
         expect { loader.add :progresss, output }.to raise_error(ArgumentError)
       end
 
+      context "with a 2nd arg defining the output" do
+        it "creates a file at that path and sets it as the output" do
+          loader.add('doc', path)
+          expect(loader.formatters.first.output).to be_a(File)
+          expect(loader.formatters.first.output.path).to eq(path)
+        end
+
+        it "accepts Pathname objects for file paths" do
+          pathname = Pathname.new(path)
```

Comment:
Do we document that `Pathname` objects are supported?  I see `IO` and `String` in our docs but hadn't noticed `Pathname`.

Also, I guess this means if you use my `case`-based implementation for the `file_at` method you'll have to make it handle `Pathname`, too.